### ISSUE TITLE
Python 3 error, builtins.TypeError: Cannot create a consistent method resolution order (MRO) for bases FancyEqMixin, object

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8499]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8499|
|Type|enhancement|
|Created|2016-06-24 20:19:39Z|
|Branch|https://github.com/twisted/twisted/tree/8499-rodrigc-mro|

When trying to run more of the tests under Python 3, I got this error

```
Traceback (most recent call last):
  File "/Users/crodrigues/twisted7/build/py36-tests-posix/lib/python3.6/site-packages/twisted/trial/runner.py", line 511, in loadPackage
    module = modinfo.load()
  File "/Users/crodrigues/twisted7/build/py36-tests-posix/lib/python3.6/site-packages/twisted/python/modules.py", line 389, in load
    return self.pathEntry.pythonPath.moduleLoader(self.name)
  File "/Users/crodrigues/twisted7/build/py36-tests-posix/lib/python3.6/site-packages/twisted/python/reflect.py", line 300, in namedAny
    topLevelPackage = _importAndCheckStack(trialname)
  File "/Users/crodrigues/twisted7/build/py36-tests-posix/lib/python3.6/site-packages/twisted/python/reflect.py", line 239, in _importAndCheckStack
    return __import__(importName)
  File "/Users/crodrigues/twisted7/build/py36-tests-posix/lib/python3.6/site-packages/twisted/conch/test/test_helper.py", line 5, in <module>
    from twisted.conch.insults import helper
  File "/Users/crodrigues/twisted7/build/py36-tests-posix/lib/python3.6/site-packages/twisted/conch/insults/helper.py", line 18, in <module>
    from twisted.python import log, _textattributes
  File "/Users/crodrigues/twisted7/build/py36-tests-posix/lib/python3.6/site-packages/twisted/python/_textattributes.py", line 29, in <module>
    class _Attribute(object, FancyEqMixin):
builtins.TypeError: Cannot create a consistent method resolution
order (MRO) for bases FancyEqMixin, object
```

<details><summary>Searchable metadata</summary>

```
trac-id__8499 8499
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8499_rodrigc_mro 8499-rodrigc-mro
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1466799579718256 1466799579718256
changetime__1466965790325539 1466965790325539
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/215

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Looks good. Thanks. Please sync with master and merge. Thanks!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset a51a0c818bfa066697ce2be53d39c1d874cdf031
```
#!CommitTicketReference repository="" revision="a51a0c818bfa066697ce2be53d39c1d874cdf031"
Merge 8499-rodrigc-mro: Fix resolution order for in _textattributes for Python 3

Author: rodrigc
Reviewer: adiroiban, alex
Fixes: #8499
```
